### PR TITLE
Bug fix and French language corrections

### PR DESCRIPTION
**Pull request by DebbieSan**
**What issue does this PR close?**

There is not any PR associated with this issue.

I have translated the "Trending Books" portion of the French home page. The intention of this translation is to ensure the home page is read more naturally by French speakers.

In addition, upon updating the file, I ran into an issue as it seems another portion of the file had been changed but not updated.

```
openlibrary/i18n/fr/messages.po:5326: 'Voir %(count)s édition'
    incompatible format for placeholder 'count': 's' and 'd' are not compatible
    Failed custom string format validation

Validation failed...
Please correct the errors before proceeding.
Validation failed.
2 errors found...
```

This was fixed by switching "s" to "d" on the French translated portion. Once updated all tests passed.

 **What does this PR achieve?**

Translates the " Trending Books" portion of the French home page.

**Testing**

To check this has been rectified on the main page, the "Trending Books" section should read " Les livres les plus populaires".

### CODE REVIEW

File: openlibrary/i18n/fr/messages.po [hunk 1]
```diff
@@ -245,12 +255,17 @@ msgstr ""
...
+#: lib/header_dropdown.html:59 lib/nav_head.html:118 login.html:10
+#: login.html:75
 msgid "Log In"
 msgstr "Se connecter"
 
-#: login.html:15
+#: account/create.html:19 login.html:16
+#, fuzzy
+msgid "OR"
+msgstr "ou"
```
Comment (cclauss) [inline openlibrary/i18n/fr/messages.po:266]: @cdrini Do we really want to modify `login-big-or`?
* https://github.com/internetarchive/openlibrary/blob/master/openlibrary/templates/login.html#L12

File: openlibrary/i18n/fr/messages.po [hunk 2]
```diff
@@ -619,16 +639,83 @@ msgstr "Quelle page regardiez vous ?"
...
+#: books/edit/addfield.html:133 books/edit/addfield.html:177 covers/add.html:79
+#: covers/manage.html:52 databarAuthor.html:66 databarEdit.html:13
+#: merge/authors.html:109 support.html:71
 msgid "Cancel"
 msgstr "Annuler"
 
+#: trending.html:10
+#, fuzzy
+msgid "Now"
+msgstr "non"
```
Comment (cclauss) [inline openlibrary/i18n/fr/messages.po:655]: Is this the correct translation?

**Review by cclauss — CHANGES_REQUESTED** [review]
Merci!!

**Comment by DebbieSan** [comment]
@cclauss oh wow, I didn't realize there were so many changes. Thank you so much for pointing this out. I had just meant to add a translation for "Trending Books", the other translations are not mine and I didn't go through the whole file yet. I do intend to do so slowly, page by page.

I just started looking at the home page and translating the items that were showing up in English. I submitted the " Trending Books" one because it was on top of the page and I ran into validation issues so I wanted to get that rectified before submitting more translations.

To clarify, I had to fix the error that was stopping the validation from completing successfully, but I see there are a lot of changes. I think the validation script must have reordered things and created a lot of changes.

File: openlibrary/i18n/fr/messages.po (hunk 2 above)
Comment (DebbieSan) [reply openlibrary/i18n/fr/messages.po:655]: No, this is definitely not the correct translation.

My suggestions would be "maintenant" or " en ce moment". 

Thank you :)

File: openlibrary/i18n/fr/messages.po [hunk 3]
```diff
@@ -387,29 +402,34 @@ msgstr "Entrée MARC non valide."
...
+#: work_search.html:80
 msgid "Times"
 msgstr "Périodes"
 
-#: merge/authors.html:89 publishers/view.html:17 subjects.html:19
-#: type/author/view.html:110
+#: subjects.html:19
+#, fuzzy
+msgid "See all works"
+msgstr "Tout voir"
```
[the remark this answers is not in the document]
Comment (DebbieSan) [reply openlibrary/i18n/fr/messages.po:429]: I would suggest either :

(Voir)  tous les livres or
(Voir)  toutes les œuvres

[the remark this answers is not in the document]
Comment (cclauss) [reply openlibrary/i18n/fr/messages.po:429]: ✅ œuvres

File: openlibrary/i18n/fr/messages.po (hunk 2 above)
Comment (cclauss) [reply openlibrary/i18n/fr/messages.po:655]: ✅ maintenant

**Comment by cdrini** [comment]
Sorry for the confusion, folks! Things labelled with `#, fuzzy` are _not_ used, so do not need to be correct. They are pairings automatically determined by the system as suggestions

File: openlibrary/i18n/fr/messages.po [hunk 4]
```diff
@@ -672,338 +753,298 @@ msgstr "Réserver"
...
-#: account/books.html:24 account/books.html:69 type/user/view.html:50
+#: account/books.html:27 account/sidebar.html:24 type/user/view.html:50
 #: type/user/view.html:55
 msgid "Reading Log"
 msgstr "Journal de lecture"
 
-#: account/books.html:28 account/books.html:83
+#: account/books.html:31 lib/nav_head.html:13 lib/nav_head.html:25
+#: lib/nav_head.html:76
+#, fuzzy
```
Comment (cdrini) [inline openlibrary/i18n/fr/messages.po:937]:
```suggestion
```

File: openlibrary/i18n/fr/messages.po (hunk 1 above)
Comment (cdrini) [reply openlibrary/i18n/fr/messages.po:266]: Yep! We should translate it, it appears on the page here:
image

**Review by cdrini — APPROVED** [review]
Looks good! Passed this through podiff to see what actually changed; generating the .po file moves a bunch of things around, making the diff look very large! But this all looks good :) Thank you @DebbieSan !

```diff
2602c2602,2603
< msgstr ""
---
> msgstr "Voir ce que les lecteurs de cette communauté ajoutent à leur "
> "bibliothèque"
2902c2903
< msgstr ""
---
> msgstr "Cette année"
3103,3104c3104,3105
< msgid "Trending"
< msgstr ""
---
> msgid "Trending Books"
> msgstr "Les livres les plus populaires"
```

^ The diff isn't looking quite right, I'm not sure why. Sigh .po files!

**Pull request merged**